src/examples/Examples.js

```diff
@@ -84,23 +88,54 @@ var Examples = React.createClass({
         <Box pad={{horizontal: 'large'}}><p><strong>Large Marquee, light text</strong></p></Box>
         {this._loremIpsum()}
         <Box pad={{ horizontal: 'large', vertical: 'none' }}>
-          <Callout
-            thumbnail="/docs/img/Video_image.png"
-            eyebrow="Video - 4:27"
-            heading="Foundation Paraguay Empowers Microbusinesses"
-            description="See how Hewlett Packard Enterprise delivers mobile solutions to improve quality of life and help eliminate poverty in South America."
-            link="#"
-            linkIcon={<PlayIcon />}
-            linkText="Watch Now"
-          />
-          <Callout
-            thumbnail="/docs/img/Case_Study_image.png"
-            eyebrow="Case Study"
-            heading="The Key Steps to Reducing Software Spend"
-            description="HPE Software Licensing and Management Solutions can help you optimize your software investments through control of complex negotiations and renewal processes"
-            link="#"
-          />
+          <Accordion
+            colorIndex="light-2"
+            headline="Services Portfolio"
+            subHeadline="Lorem ipsum dolor sit amet, dicat sonet congue ei mei, est
```

Comment:
I usually prefer it on one line especially because it's HTML markup (for some of the cases) and that  I usually don't want to introduce invisible characters unnecessarily that can mess with the rendered output (possibly not so much in this case).